code/modules/robotics/robot/module_tool_creator/modules.dm

```diff
@@ -13,15 +13,16 @@
 		/obj/item/noisemaker,
 		/obj/item/robot_foodsynthesizer,
 		/obj/item/reagent_containers/food/drinks/bottle/beer/borg,
-		/obj/item/reagent_containers/food/drinks/drinkingglass,
+		/obj/item/reagent_containers/food/drinks/drinkingglass/pitcher,
+		/obj/item/pen/crayon/random/robot,
 		/obj/item/coin_bot,
-		/obj/item/dice_bot,
+		/obj/item/dice/robot,
 		/obj/item/device/light/zippo/borg,
 		/obj/item/pen, // TODO: make more versatile version
 		/obj/item/device/prisoner_scanner,
 		/obj/item/item_box/assorted/stickers/robot,
 		// TODO: security grenade fabricator?!
-		// TODO: nerfed/limited cuffs/zip-ties?
+		/obj/item/handcuffs/tape_roll/crappy,
```

Comment:
Unsure how okay I am with this, esp. given that they can now more easily manage some knock-out chems with their second chem container (pitcher + glass bottle).

I feel like this is going to cause more trouble than its worth, looking for second opinions on it.

Reply:
This can definitely be easily removed if needed.

Reply:
Sure, let's go with removing the inclusion of the tool for now, but keep the tool's definition in so that it can easily be added later (admins can add tools to cyborgs mid-round, for example). I might encourage them to try adding them to the relevant cyborgs during rounds they're watching and seeing how they behave with it, for example.